src/mongo/delegates/Views.class.php

```diff
@@ -822,24 +869,6 @@ protected function extractProperties($source,$viewSpec,$from)
     }
 
     /**
-     * @param string $viewSpec
-     * @return string
-     */
-    private function getFromCollectionForViewSpec($viewSpec)
```

Comment:
The validation says that `"from"` is a mandatory field on view specifications, so there is no eventuality when we should fall back on `$this->podName`.  Removing, in favour of fetching the `'from'` collection name from the spec directly.